maia-hdl/projects/pluto/system_bd.tcl

```diff
@@ -262,29 +270,410 @@ ad_connect  maia_sdr_clk/clk_out3 maia_sdr/clk3x_clk
 ad_connect  sys_cpu_clk maia_sdr_clk/clk_in1
 ad_connect  sys_cpu_reset maia_sdr_clk/reset
 
+if {[info exists maia_iio]} {
+
+	ad_ip_instance axi_dmac axi_ad9361_dac_dma
+	ad_ip_parameter axi_ad9361_dac_dma CONFIG.DMA_TYPE_SRC 0
+	ad_ip_parameter axi_ad9361_dac_dma CONFIG.DMA_TYPE_DEST 1
+	ad_ip_parameter axi_ad9361_dac_dma CONFIG.CYCLIC 1
+	ad_ip_parameter axi_ad9361_dac_dma CONFIG.AXI_SLICE_SRC 0
+	ad_ip_parameter axi_ad9361_dac_dma CONFIG.AXI_SLICE_DEST 0
+	ad_ip_parameter axi_ad9361_dac_dma CONFIG.DMA_2D_TRANSFER 0
+	ad_ip_parameter axi_ad9361_dac_dma CONFIG.DMA_DATA_WIDTH_DEST 64
+
+	ad_ip_instance util_upack2 tx_upack
+
+	ad_ip_instance axi_dmac axi_ad9361_adc_dma
+	ad_ip_parameter axi_ad9361_adc_dma CONFIG.DMA_TYPE_SRC 2
+	ad_ip_parameter axi_ad9361_adc_dma CONFIG.DMA_TYPE_DEST 0
+	ad_ip_parameter axi_ad9361_adc_dma CONFIG.CYCLIC 0
+	ad_ip_parameter axi_ad9361_adc_dma CONFIG.SYNC_TRANSFER_START 0
+	ad_ip_parameter axi_ad9361_adc_dma CONFIG.AXI_SLICE_SRC 0
+	ad_ip_parameter axi_ad9361_adc_dma CONFIG.AXI_SLICE_DEST 0
+	ad_ip_parameter axi_ad9361_adc_dma CONFIG.DMA_2D_TRANSFER 0
+	ad_ip_parameter axi_ad9361_adc_dma CONFIG.DMA_DATA_WIDTH_SRC 64
+	ad_ip_instance util_cpack2 cpack
+
+	ad_connect axi_ad9361/adc_enable_i0 cpack/enable_0
+	ad_connect axi_ad9361/adc_data_q0 cpack/fifo_wr_data_1
+
+	ad_connect axi_ad9361/l_clk cpack/clk
+	ad_connect axi_ad9361/rst cpack/reset
+
+
+
```

Comment:
Here and below leave at most one empty line between code blocks (there are 3 blank lines here).